Dangerfile

```diff
@@ -1,46 +1,31 @@
 # frozen_string_literal: true
 
-def wip_feature?
-  has_wip_label = github.pr_labels.any? { |label| label.include?('WIP') }
-  has_wip_title = github.pr_title.include?('WIP')
+github.dismiss_out_of_range_messages
 
-  has_wip_label || has_wip_title
-end
+# `files: []` forces rubocop to scan all files, not just the ones modified in the PR
+rubocop.lint(files: [], force_exclusion: true, inline_comment: true, fail_on_inline_comment: true, include_cop_names: true)
 
-# these are reviewers actively providing feedback and potentially changing the state of the PR (approved, changes-requested)
-def active_reviewers?
-  repo_name = github.pr_json['base']['repo']['full_name']
-  pr_number = github.pr_json['number']
+manifest_pr_checker.check_all_manifest_lock_updated
 
-  !github.api.pull_request_reviews(repo_name, pr_number).empty?
+# skip remaining checks if we're during the release process
+if github.pr_labels.include?('Releases')
+  message('This PR has the `Releases` label: some checks will be skipped.')
+  return
 end
 
-# requested_teams / requested_reviewers are users initially requested to review a PR, who haven't reacted yet
-def requested_reviewers?
-  has_requested_reviews = !github.pr_json['requested_teams'].to_a.empty? || !github.pr_json['requested_reviewers'].to_a.empty?
-  has_requested_reviews || active_reviewers?
-end
+view_changes_checker.check
 
-return if github.pr_labels.include?('Releases')
+pr_size_checker.check_diff_size(max_size: 500)
 
-github.dismiss_out_of_range_messages
-
-manifest_pr_checker.check_all_manifest_lock_updated
+# skip remaining checks if we have a Draft PR
+return if github.pr_draft?
 
 labels_checker.check(
+  do_not_merge_labels: ['Do Not Merge'],
   required_labels: [//],
   required_labels_error: 'PR requires at least one label.'
 )
 
-view_changes_need_screenshots.view_changes_need_screenshots
-
-pr_size_checker.check_diff_size
-
 milestone_checker.check_milestone_due_date(days_before_due: 2)
 
-# `files: []` forces rubocop to scan all files, not just the ones modified in the PR
-rubocop.lint(files: [], inline_comment: true, fail_on_inline_comment: true, include_cop_names: true)
-
-warn('PR is classed as Work in Progress') if wip_feature?
-
-warn("No reviewers have been set for this PR yet. Please request a review from **@\u2028Automattic/pocket-casts-ios**.") unless requested_reviewers?
+warn('PR is classed as Work in Progress') if github_utils.wip_feature?
```

Comment:
Another Dangermattic-level suggestion. I like `github_utils.wip_feature?`. Have you thought about implementing a similar `github_utils.release_pr?` syntax-sugar/helper method?

Reply:
Thanks! I've added it to the same item in backlog I [mentioned above](https://github.com/orgs/Automattic/projects/775?pane=issue&itemId=53114939).